python/taichi/lang/__init__.py

```diff
@@ -398,6 +398,7 @@ def __init__(self):
         self.gdb_trigger = False
         self.experimental_real_function = False
         self.short_circuit_operators = False
+        self.ndarray_use_torch = False
```

Comment:
Hmm this is not the best place for ndarray_use_torch, since it's going away soon I'm fine with this as a temporary location. :D

Reply:
ok~